manager/manager.go

```diff
@@ -108,81 +105,79 @@ func (m *Manager) Close() (err error) {
 	return m.transactions.Close()
 }
 
-// New creates a transaction for the given config
-func (m *Manager) New(config transaction.Config) (tx transaction.Transaction, err error) {
-
-	d, ok := m.drivers.Load(config.Driver)
+// Run the given transaction
+func (m *Manager) Run(c transaction.Config) (r transaction.Result) {
+	var err error
 
-	if !ok {
-		return nil, fmt.Errorf("manager: transaction driver %s not registered", config.Driver)
-	}
+	uuid := ksuid.New().String()
+	start := time.Now()
 
-	if config.Inputs != nil {
-		if config, err = tmpl.Parse(config); err != nil {
-			return nil, fmt.Errorf("manager: error parsing transaction template: %w", err)
+	if c.Inputs != nil {
+		if c, err = tmpl.Parse(c); err != nil {
+			return wrapErrorResult(
+				uuid, c, start, fmt.Errorf("manager: error parsing transaction template: %w", err))
 		}
 	}
 
-	if config.Timeout == "" {
-		config.Timeout = DefaultTransactionTimeout
+	buf, err := json.Marshal(&c)
+	if err != nil {
+		return wrapErrorResult(
+			uuid, c, start, fmt.Errorf("manager: error marshaling config: %w", err))
 	}
 
-	return d.(driver.Driver).New(config)
-}
+	req, err := http.NewRequest(http.MethodPost, m.executorURL+uuid, bytes.NewReader(buf))
```

Comment:
Are you taking into account the prefix that might be set in the executor? Or is this supposed to be the URL with the prefix already attached to it?

Reply:
The prefix is something i was willing to kill, at least for now. It is not exposed anymore.